pkg/odo/cli/init/init.go

```diff
@@ -49,6 +49,9 @@ var initExample = templates.Examples(`
   # Bootstrap a new component with a specific versioned devfile from registry
   %[1]s --name my-app --devfile nodejs --devfile-version 2.1.0
 
+ # Bootstrap a new component with the latest devfile from registry
```

Comment:
In the previous example:
```
%[1]s --name my-app --devfile nodejs
```

you could indicate that if several versions exist, the one marked as `default` will be used